Makefile

```diff
@@ -38,7 +38,7 @@ TEKTON_TRIGGERS_VERSION ?= latest
 TEKTON_DASHBOARD_VERSION ?= latest
 TEKTON_RESULTS_VERSION ?= v0.4.0 # latest returns an older version hence hard coding to v0.3.1 for now (tektoncd/results#138)
 PAC_VERSION ?= 0.5.5
-TEKTON_HUB_VERSION ?= v1.7.1 # latest doesn't returns any version hence hard coding to v1.7.1 for now
+TEKTON_HUB_VERSION ?= v1.7.2 # latest doesn't returns any version hence hard coding to v1.7.1 for now
```

Comment:
the purpose of `fetch-release` script is to avoid PRs like this.

ie, in the main branch the reference should be latest or nightly.
Then during a release branch creation we can freeze a version.
This is how all other components are managed.

This process of updating version after each release is not easy to maintain.

Could we make a url with `latest` or `nightly` available for tektonhub and use that in main branch.

Reply:
can we do something like https://github.com/tektoncd/operator/pull/740

Reply:
We definitely can 👼🏼

Reply:
@piyush-garg @PuneetPunamiya this will need to be cherry-picked in 0.55.x ?

Reply:
yes this will need a cherry-pick on 0.55.x, will create once this patch is merged

Reply:
@nikhil-thomas I agree with your comments, can we work on the feature of having the latest here and have the param in the release branch script in a follow-up PR?